core/commonMain/src/ArgParser.kt

```diff
@@ -73,14 +85,18 @@ class ArgParserResult(val commandName: String)
 /**
  * Arguments parser.
  *
- * @property programName name of current program.
- * @property useDefaultHelpShortName add or not -h flag for help message.
- * @property prefixStyle style of expected options prefix.
- * @property skipExtraArguments just skip extra arguments in command line string without producing error message.
+ * @property programName the name of the current program.
+ * @property useDefaultHelpShortName specifies whether to register "-h" option for printing the usage information.
+ * @property prefixStyle the style of option prefixing.
+ * @property skipExtraArguments specifies whether the extra arguments in a command line string
```

Comment:
May be "specifies whether the extra arguments in a command line string are skipped(or legal)"?

Reply:
"can be skipped without producing an error message" impies "legal", no?

Reply:
that's ok